src/components/Nav.js

```diff
@@ -6,33 +6,31 @@ class Nav extends Component {
   render () {
     return (
       <div>
-        <div>
-          <div className='header-text'>
-            <Link to='/'>
-              Adopt A Family
-            </Link>
+        <div className='header-text'>
+          <Link to='/'>
+            Adopt A Family
+          </Link>
+        </div>
+        <div className="header-links">
+          <div className="header-mission">
+            Mission
```

Comment:
Let's get these set up as place holder links since we don't have any of these pages set up yet.